src/popup/router/components/Filters.vue

```diff
@@ -35,15 +35,28 @@ export default {
   props: {
     value: { type: Object, required: true },
     filters: { type: Object, required: true },
-    fixed: Boolean,
+    sticky: Boolean,
+    withSearch: Boolean,
   },
   data() {
     return {
       rotatableFilters: this.filters,
     };
   },
+  computed: {
+    cssVars() {
+      return this.withSearch ? {
+        '--header-height': '116px',
+        '--account-info-height': '62px',
+      } : {
+        '--header-height': '104px',
+        '--account-info-height': '62px',
+      };
+    },
+  },
   methods: {
     handleClick([filter, { rotated = null }]) {
+      document.querySelector('.app-inner').scrollTo({ top: 140, behavior: 'smooth' });
```

Comment:
I strongly recommend to not make the components dependant on the parent structure. Also please avoid magic numbers.